### DISCUSSION TITLE
Thanks for all of your effort.

### DISCUSSION THREAD

**Discussion by AmplificPanda** [ClassicOldSong/Apollo#62] [General]
Hey man, just wanted to write a thank you message to you for this project, I have had so many people over for 4 player local games through NVIDIA Shield + Moonlight + Apollo. Your Sunshine fork is by far the best and is 99.99% stutter free for me.  Infinitely better than the default Sunshine application. 

Please let me know if there are any ways I can popularise your repository for this.

**Comment by ClassicOldSong** [comment]
Thanks for your appreciation, simply sharing this with people who have needs for streaming would be enough!
